integration-tests/mail/src/main/resources/application.properties

```diff
@@ -14,13 +14,13 @@
 ## See the License for the specific language governing permissions and
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
+
 #
 # Quarkus
 #
 quarkus.ssl.native=true
-quarkus.log.file.enable = false
 
 #
 # Camel
 #
-camel.context.name = quarkus-camel-example
\ No newline at end of file
+camel.context.name = quarkus-camel-example
```

Comment:
`camel.context.name` is another good candidate for removal in the next cleanup round